src/griffe/agents/inspector.py

```diff
@@ -287,7 +287,11 @@ def inspect_class(self, node: ObjectNode) -> None:
         Parameters:
             node: The node to inspect.
         """
-        bases = [base.__name__ for base in node.obj.__bases__ if base is not object]
+        bases = []
+        for base in node.obj.__bases__:
+            if base is object:
+                continue
+            bases.append(f"{base.__module__}.{base.__name__}")
```

Comment:
We need the fully qualified name to resolve the bases later, in `Class.resolved_bases`.

Reply:
You're right, good catch. I think I can re-use the object node (`node` here) to build the path of the object, since every node has a name and a pointer to its parent.

Reply:
Solved this by using the class' `__qualname__` attribute instead.